### PR TITLE
projects-todos, Katharina Liebig

### PR DESCRIPTION
**Pull request by katliebig**
Link to my deployed project: https://tasks-to-do-project-todos.netlify.app/

My reflections on how this week's project turned out: I'm happy with how the project turned out, especially after refactoring parts of the code.

Things I'd like to have clarified or explained in more detail: -

### CODE REVIEW

File: code/src/components/HeaderFilter.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+`
+
+const HeaderFilter = () => {
+  const dispatch = useDispatch()
+
+  const onFilterChange = (filter) => {
+    dispatch(tasks.actions.changeFilter(filter))
+  }
+
+  return (
```
Comment (AsaHildebrand) [inline code/src/components/HeaderFilter.js:56]: Very nice with all the filters, makes it very easy to find which tasks you want to see 😃

File: code/src/components/ActionComplete.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+const Icon = styled.img`
+  width: 12px;
+`
+
+const ActionComplete = () => {
+  const dispatch = useDispatch()
+
+  const handleClick = () => {
+    const completeAll = window.confirm("Are you sure you want to complete all tasks?")
```
Comment (AsaHildebrand) [inline code/src/components/ActionComplete.js:34]: Nice that this question appears when you click on Complete all button 👍

File: code/src/components/ActionDelete.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+const Icon = styled.img`
+  width: 12px;
+`
+
+const ActionDelete = () => {
+  const dispatch = useDispatch()
+
+  const handleClick = () => {
+    const deleteAll = window.confirm("Are you sure you want to delete all tasks?")
```
Comment (AsaHildebrand) [inline code/src/components/ActionDelete.js:34]: 👍

File: code/src/components/Task.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+`
+const Icon = styled.img`
+  width: 24px;
+  cursor: pointer;
+`
+const IconBin = styled.img`
+  width: 20px;
+  cursor: pointer;
+`
+const TaskTitle = styled.p`
```
Comment (AsaHildebrand) [inline code/src/components/Task.js:30]: And same here 😃

File: code/src/components/Task.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  background-color: transparent;
+  padding: 5px 10px;
+  display: flex;
+
+`
+const Icon = styled.img`
+  width: 24px;
+  cursor: pointer;
+`
+const IconBin = styled.img`
```
Comment (AsaHildebrand) [inline code/src/components/Task.js:26]: Maybe add an empty line here 😃

**Review by AsaHildebrand — APPROVED** [review]
Very nice project, I would gladly use your todo-list every day :D
All of the code is very easy to read, and I think it is easy to follow and understand how it is working even though I couldn't have written it by my self 🥇 I have nothing to add, except for two empty lines. 😄 Really good job 🥰

**Review by KarolCislowski — APPROVED** [review]
Great app 🌈 I really like all the extra functionalities you created here 💯
The next step would be to create the desktop layout which wouldn't fill like mobile laying on the screen 😸

**Pull request closed without merging**